product-service/db/queries.ts

```diff
@@ -0,0 +1,17 @@
+export const selectAllProducts = `select id, title, description, price, count from products p left join stock s on s.product_id = p.id;`;
+export const selectProductById = id => `select id, title, description, price, count from products p left join stock s on s.product_id = p.id where (p.id = '${id}');`;
+export const addNewProduct = ({ title, description, count, price }) => `
+  with data(title, description, count, price) as (
+    values
+      ('${title}', '${description}', ${count}, ${price})
```

Comment:
In my opinion, this method of generating queries is an antipattern. It is best to read the documentation on what are parameterized queries by reference https://node-postgres.com/features/queries